service/awsconfig/v10/resource/cloudformation/validate_test.go

```diff
@@ -67,17 +69,17 @@ func Test_validateHostPeeringRoutes(t *testing.T) {
 
 			newResource, err = New(c)
 			if err != nil {
-				t.Error("expected", nil, "got", err)
+				t.Fatal("expected", nil, "got", err)
```

Comment:
This was a ball ache again. The tests do not really say what is going on when only `Error` is used. It fails silently, goes ahead, and fails at some point and you start digging. This is obvious now.

Reply:
Yes I've had the same problem and I think using Fatal makes it easier to debug. I think changing this when we modify the code is good.